plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -479,15 +534,30 @@ void toIndex(FileIndex[] files, Element root, string htmlFileDir) @trusted {
             "filter_table_on_search('fileFilterInput', 'fileTable')").addClass(
             "form-control").setAttribute("placeholder", "Search...");
 
-    auto tbl = tmplSortableTable(root, [
-            "Path", "Score", "Alive", "NoMut", "Total", "Time (min)"
-            ]);
+    Table tbl;
```

Comment:
line 537 to 548 is initialization of tbl. I recommend using a lambda for the init.

```d
Table tbl = () { if (x) return tbl.init else return tblAwesome; }();
```